maxim/tests/test_openai.py

```
@@ -33,70 +25,168 @@ async def asyncSetUp(self):
-        async for chunk in response:
-            print(chunk.choices[0].delta.content or "", end="")
+        print("Response: ", response)
+        self.logger.flush()
+
+    async def test_async_chat_completions_error_handling(self):
+        client = MaximOpenAIClient(
+            OpenAI(api_key=openaiApiKey), logger=self.logger
+        ).aio
+
+        # Test invalid model
+        with self.assertRaises(Exception):
```

Ruff (0.12.2)

Use `pytest.raises` instead of unittest-style `assertRaises`

Replace `assertRaises` with `pytest.raises`

(PT027)